### PR TITLE
Add descriptions for sections in assessment reports

### PR DESCRIPTION
**Pull request by makalaaneesh**

### CODE REVIEW

File: yb-voyager/src/utils/commonVariables.go [hunk 1]
```diff
@@ -78,11 +78,12 @@ type SchemaReport struct {
...
-	SchemaNames []string   `json:"SchemaNames,omitempty"`
-	DBVersion   string     `json:"DbVersion,omitempty"`
-	Notes       []string   `json:"Notes,omitempty"`
-	DBObjects   []DBObject `json:"DatabaseObjects"`
+	DBName        string     `json:"DbName,omitempty"`
+	SchemaNames   []string   `json:"SchemaNames,omitempty"`
+	DBVersion     string     `json:"DbVersion,omitempty"`
+	Notes         []string   `json:"Notes,omitempty"`
+	DBObjects     []DBObject `json:"DatabaseObjects"`
+	DBObjectsDesc string     `json:"DatabaseObjectsDesc,omitempty"`
```
Comment (sanyamsinghal) [inline yb-voyager/src/utils/commonVariables.go:86]: changing this struct will also reflect in the analyze schema report, right?

how about adding it in the `assessmentReport` struct as: `assessmentReportSchemaSummaryDesc`

**Comment by makalaaneesh** [comment]
https://jenkins.dev.yugabyte.com/job/users/job/yb-voyager-testing/job/yb-voyager-testing-pipeline/2835/

File: yb-voyager/cmd/templates/assessmentReport.template [hunk 2]
```diff
@@ -75,6 +75,7 @@
         {{end}}
 
         <h2>Database Objects</h2>
+        <p>{{.SchemaSummary.DBObjectsDesc}}</p>
```
Comment (sanyamsinghal) [inline yb-voyager/cmd/templates/assessmentReport.template:78]: minor miss, rename `.SchemaSummary.DBObjectsDesc` -> `.SchemaSummaryDBObjectsDesc`
Will fix it in my ongoing PR
cc @makalaaneesh

Comment (makalaaneesh) [reply yb-voyager/cmd/templates/assessmentReport.template:78]: 🙏

**Pull request merged**